### PR TITLE
docs(oidc): add freshrss oidc integration guide

### PR DESCRIPTION
**Pull request by darkpixelftw**
Create integration guide for oidc authentication for freshrss and authelia

Document non-obvious behaviour that freshrss expects port in the redirect URI

Signed-off-by: darkpixelftw <darkpixelftw@outlook.com>

### CODE REVIEW

**Comment by james-d-elliott** [comment]
Really appreciate the contribution! If you have time I'd appreciate confirming the token endpoint auth method this client uses. We will likely be requiring this to be configured in authelia in the next release.

**Comment by darkpixelftw** [comment]
Hi, I asked the maintainer of FreshRSS what the token auth method is and it seems we're having a bit of trouble answering this, see: https://github.com/FreshRSS/FreshRSS/issues/6102

**Comment by james-d-elliott** [comment]
Thanks, it's `client_secret_basic`. I'll make the recommendations based on this.

File: docs/content/en/integration/openid-connect/freshrss/index.md
Comment (james-d-elliott) [inline docs/content/en/integration/openid-connect/freshrss/index.md:19]:
```suggestion
  * [v4.38.0](https://github.com/authelia/authelia/releases/tag/v4.38.0)
```

File: docs/content/en/integration/openid-connect/freshrss/index.md
Comment (james-d-elliott) [inline docs/content/en/integration/openid-connect/freshrss/index.md:64]:
```suggestion
      token_endpoint_auth_method: 'client_secret_post'
      userinfo_signed_response_alg: 'none'
```

File: docs/content/en/integration/openid-connect/freshrss/index.md
Comment (james-d-elliott) [inline docs/content/en/integration/openid-connect/freshrss/index.md:63]:
```suggestion
        - 'openid'
        - 'groups'
        - 'email'
        - 'profile'
```

File: docs/content/en/integration/openid-connect/freshrss/index.md
Comment (james-d-elliott) [inline docs/content/en/integration/openid-connect/freshrss/index.md:58]:
```suggestion
        - 'https://freshrss.example.com:443/i/oidc/'
```

File: docs/content/en/integration/openid-connect/freshrss/index.md
Comment (james-d-elliott) [inline docs/content/en/integration/openid-connect/freshrss/index.md:56]:
```suggestion
      authorization_policy: 'two_factor'
```

File: docs/content/en/integration/openid-connect/freshrss/index.md
Comment (james-d-elliott) [inline docs/content/en/integration/openid-connect/freshrss/index.md:53]:
```suggestion
    - id: 'freshrss'
      description: 'freshrss'
```

File: docs/content/en/integration/openid-connect/freshrss/index.md
Comment (james-d-elliott) [inline docs/content/en/integration/openid-connect/freshrss/index.md:37]:
```suggestion

### Special Notes

1. The FressRSS implementation seems to always include the port in the requested `redirect_uri`. As Authelia strictly conforms to the specifications this means the client registration **_MUST_** include the port for the requested `redirect_uri` to match.

```

**Review by james-d-elliott — CHANGES_REQUESTED** [review]
Thanks so much for contributing! Just a few changes. I know you've not tested with v4.38.0 but we're only accepting PR's with this version in mind as we're pretty close to a release.

If you'd like to verify this I can probably help you. Otherwise we can wait for the release and you can reevaluate.

**Comment by darkpixelftw** [comment]
> Thanks so much for contributing! Just a few changes. I know you've not tested with v4.38.0 but we're only accepting PR's with this version in mind as we're pretty close to a release.
> 
> If you'd like to verify this I can probably help you. Otherwise we can wait for the release and you can reevaluate.

Sure it would be a great help if you could verify it on the 4.38 version, If not then I'll just wait for the release. Is there an ETA for when the next release is?

File: docs/content/en/integration/openid-connect/freshrss/index.md
Comment (james-d-elliott) [inline docs/content/en/integration/openid-connect/freshrss/index.md:68]:
```suggestion
      token_endpoint_auth_method: 'client_secret_basic'
```

**Review by james-d-elliott — CHANGES_REQUESTED** [review]

**Comment by james-d-elliott** [comment]
We don't have a specific ETA, but it should be soon. We can try it against the beta but you'll need a database backup.

**Pull request merged**
